2020-01-20 15:20:24,546 INFO: Workout App startup [in /Users/danielvaroli/Desktop/personal_website/danielvaroli/app/__init__.py:52]
2020-01-20 15:20:31,729 ERROR: Exception on / [GET] [in /Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/app/main/routes.py", line 17, in home
    return render_template('main.html')
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/app/templates/main.html", line 10, in template
    {% enblock %}
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'enblock'. Jinja was looking for the following tags: 'endblock'. The innermost block that needs to be closed is 'block'.
2020-01-20 15:21:04,631 ERROR: Exception on / [GET] [in /Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/app/main/routes.py", line 17, in home
    return render_template('main.html')
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/app/templates/main.html", line 1, in top-level template code
    {% extends 'welcome.html' %}
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/Users/danielvaroli/Desktop/personal_website/danielvaroli/venv/lib/python3.7/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: welcome.html
